contrib/devtools/copyright_header.py

```diff
@@ -91,10 +91,10 @@ def compile_copyright_regex(copyright_style, year_style, name):
     "Satoshi Nakamoto\n",
     "The Bitcoin Core developers\n",
     "Bitcoin Core Developers\n",
-    "BitPay Inc\.\n",
-    "University of Illinois at Urbana-Champaign\.\n",
+    r"BitPay Inc\.\n",
+    r"University of Illinois at Urbana-Champaign\.\n",
     "Pieter Wuille\n",
-    "Wladimir J. van der Laan\n",
+    r"Wladimir J\. van der Laan\n",
     "Jeff Garzik\n",
```

Comment:
It seems inconsistent to have some lines without a newline, some with a '\n' and others with a '\\n'. I'd prefer to use the same suffix for all (or leave it and append the suffix at runtime)